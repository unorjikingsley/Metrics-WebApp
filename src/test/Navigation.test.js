import React from 'react';
import renderer from 'react-test-renderer';
import Nav from '../Components/Navigation';

test('should match the snapshot', () => {
  const home = renderer.create(<Nav />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});
