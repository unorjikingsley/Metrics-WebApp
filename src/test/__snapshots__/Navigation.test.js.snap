// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="header"
>
  <div
    className="header_logo"
  >
    <i
      className="fas fa-globe"
    />
    <h1>
      The Globe Trotter !
    </h1>
  </div>
  <div
    className="globe"
  >
    <h2>
      Welcome To The Globe
    </h2>
  </div>
</div>
`;
